src/Language/PureScript/Docs/Types.hs

```diff
@@ -777,8 +805,15 @@ instance A.ToJSON Declaration where
              , "sourceSpan" .= declSourceSpan
              , "children"   .= declChildren
              , "info"       .= declInfo
+             , "kind"       .= declKind
              ]
 
+instance A.ToJSON KindInfo where
+ toJSON (KindInfo kindFor ty) =
+   A.object [ "keyword" .= P.kindSignatureForKeyword kindFor
```

Comment:
I think that if this is the only place we are using `kindSignatureForKeyword`, then it should live in this module alongside the `asKindSignatureFor` parser, because that's what it needs to be kept in sync with.

Reply:
Will fix.